linkerd/docs/response_classifier.md

```diff
@@ -80,7 +80,31 @@ All responses are considered to be successful, regardless of status code.
 
 # gRPC Response Classifiers
 
-For HTTP/2 routers that handle gRPC traffic, four additional response classifiers are available to categorize responses based on [gRPC status codes](https://github.com/grpc/grpc/blob/master/doc/statuscodes.md) in the stream's trailers frame. Status code 0 (`OK`) is always considered successful, while all other gRPC status codes are considered failures.
+For HTTP/2 routers that handle gRPC traffic, one optional parameter and four
+additional response classifiers are available to categorize responses based on
+[gRPC status codes](https://github.com/grpc/grpc/blob/master/doc/statuscodes.md)
+in the stream's trailers frame.
+
+By default, status code 0 (`OK`) is always considered successful, while all
+other gRPC status codes are considered failures. There exists an optional
+parameter `nonAccruableStatusCodes` that accepts a user-defined list of error
+status codes that should not count towards the total failure count for a
+router.
+
+```yaml
+routers:
+- protocol: h2
+  experimental: true
+  service:
+    responseClassifier:
+      kind: io.l5d.h2.grpc.default
+      nonAccruableStatusCodes:
+      - 3
+      - 5
+```
+
+Status code 3 (`InvalidArgument`) and 5 (`NotFound`) will not be counted
```

Comment:
I believe that status code 0 will also be counted as a success, which is a bit confusing because it doesn't appear in this list.  What do you think of removing the special casing for 0?  i.e. making it so ONLY status codes in this list are counted as success (but making `[0]` the default value for this property)